includes/polling/mempools/stoneos.inc.php

```diff
@@ -0,0 +1,5 @@
+<?php
+echo 'Pool StoneOs memory';
+$mempool['total'] = (snmp_get($device, 'HILLSTONE-SYSTEM-MIB::sysTotalMemory.0', '-OvQU') * 1024);
```

Comment:
Please switch to using `snmp_get_multi_oid` instead of separate `snmp_get` calls. Also, pass the MIB name rather than `HILLSTONE-SYSTEM-MIB::` as before.